src/Api/ApiSingleSignOn.php

```diff
@@ -72,18 +72,30 @@ public function viewId()
 
     public function url()
     {
-        $data = [
-            'key' => $this->key->key,
-        ];
+        $data = $this->urlHandler();
+        return $this->api->url('auth/sso', $data);
+    }
 
-        if ($type = $this->viewType()) {
-            $data['view_type'] = $type;
-        }
+    public function embeddedUrl()
```

Comment:
it would be better to have only one URL method but add a `embed()` method which just sets an `embed = true` flag which the URL method checks (much like we have viewType and viewId in the url method). So the syntax for embedding would look like this:

```
        return (new ApiSingleSignOn($apiKey))->view($server)->embed()->url();
```

This type of syntax (called "fluent" syntax) requires that each method return `$this` besides the final method that gives you a result (URL).